index.html

```diff
@@ -46,5 +46,142 @@ <h4 class="welcome">LET'S CONNECT</h4>
       </ul>
     </section>
   </main>
+  <section class="card">
+    <figure class="picture">
+      <img src="images/Portf1.png" alt="">
```

Comment:
Kindly update your README file is missing important information. If you need a template about the requirements missing you will have more about it in your **Project Requirements > General Requirements**. Template for your README file it's[ here](https://github.com/microverseinc/readme-template).

Reply:
- [ ] The issue here has not been properly resolved.

Reply:
> 
> 
>     * [ ]  The issue here has not been properly resolved.

Can You please describe that question as well. What is missing in read me file